acceptance-tests/src/test/java/tech/pegasys/web3signer/tests/HttpHostAllowListAcceptanceTest.java

```diff
@@ -27,7 +27,8 @@
 
   @Test
   void httpEndpointWithDefaultAllowHostsRespondsWithOkResponse() {
-    final SignerConfiguration signerConfiguration = new SignerConfigurationBuilder().build();
+    final SignerConfiguration signerConfiguration =
+        new SignerConfigurationBuilder().withMode("eth2").build();
```

Comment:
might be overkill and probably is, but do we want to run these test with the other modes as well?

Reply:
I think it is overkill, but only because I know how the underlying app works ...